examples/app-playground/app/server-actions/Form.tsx

```diff
@@ -0,0 +1,20 @@
+import Button from '#/ui/button';
+import { NameInput } from './components/NameInput';
+
+export function Form() {
+  async function formAction(formData: FormData) {
+    'use server';
+    console.log('Form Input: ', formData.get('name'));
```

Comment:
nit: This will cause a double space, right?

```suggestion
    console.log('Form Input:', formData.get('name'));
```

Reply:
You're right! Will resolve this and others.